ros_buildfarm/config/build_file.py

```diff
@@ -37,6 +37,8 @@ class BuildFile(object):
     def __init__(self, name, data):
         self.name = name
 
+        self.custom_rosdep_urls = []
+
```

Comment:
Can do, but will then need to change ``get_repositories_and_script_generating_key_files()`` in common.py as some build_file objects passed into it will have ``custom_rosdep_urls`` defined and some will not (because that function gets called on release, source, and doc build files). Is ``hasattr`` the best way of ensuring I don't try to access an attribute that isn't defined or is there a better way that I should use instead?